client/src/components/organisms/RequestGroupForm.tsx

```diff
@@ -264,19 +265,26 @@ const RequestGroupForm: FunctionComponent<Props> = (props: Props) => {
     /* Functions for RequestGroup's Image */
     const updateImageError = (image: string) => {
         let error = "";
-        if (image === "") {
+        if (image === "" && uploadedImg === "") {
             error = "Please select an image";
         }
 
         setImageError(error);
         return error;
     };
 
+    const onUploadImg = (uploadedImg: string) => {
+        setChangeMade(true);
+        updateImageError(uploadedImg);
+        setUploadedImg(uploadedImg);
+    };
```

Comment:
when an image is uploaded do we need to call `setImage("")`? correct me if I'm wrong but it seems like at most one of `image` and `uploadedImage` should be non-empty at any point (based on how it's conditionally rendered in `ImagePicker`

Reply:
One of them should be empty at all times but a user needs to clear the selected image before they can upload their own image so I don't think it's necessary.